lib/solid/profile.js

```diff
@@ -176,6 +194,52 @@ SolidProfile.prototype.appendFromGraph =
   }
 
 /**
+ * Returns the value of a given "field" (predicate) from the profile's parsed
+ * graph. Note: If there are more than one matches for this predicate, .find()
+ * returns the first one.
+ * Usage:
+ *
+ *   ```
+ *   var inboxUrl = profile.find(vocab.solid('inbox'))
+ *   ```
+ * @method find
+ * @param predicate {NamedNode} RDF named node of the predicate
+ * @return {String} String value (or uri)
+ */
+SolidProfile.prototype.find = function find (predicate) {
+  if (!this.parsedGraph) {
+    throw new Error('Profile graph not yet loaded.')
+  }
+  var subject = rdf.sym(this.webId)
+  var result = this.parsedGraph.any(subject, predicate)
+  return result.value || result.uri
```

Comment:
For the moment, yeah. (That's the rdflib API, until we update it to use the rdf js spec)